mobile-widgets/themeinterface.cpp

```diff
@@ -1,6 +1,9 @@
 // SPDX-License-Identifier: GPL-2.0
 #include "themeinterface.h"
+#include "qmlmanager.h"
+#include "core/metrics.h"
 #include "core/settings/qPrefDisplay.h"
+#include <QDebug>
```

Comment:
let's not use `qDebug()` [yes, I know, we do elsewhere] if this is something we want in the AppLog, then let's use the `appendTextToLog()` method in the `QMLManager`

Reply:
It is not really worth while putting in the log, so I removed it.

I know I should use  appendTextLog() I just have a problem cross connecting the UI close files. One day it should move to the backend and be shared with whatever desktop uses.